solutions/swb-reference/integration-tests/support/utils/regExpressions.ts

```diff
@@ -2,4 +2,11 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
+import { resourceTypeToKey } from '@aws/workbench-core-base';
+
 export const uuidRegExp: RegExp = /\w{8}-\w{4}-\w{4}-\w{4}-\w{12}/;
+
+// eslint-disable-next-line @rushstack/security/no-unsafe-regexp,security/detect-non-literal-regexp
+export const envUuidRegExp: RegExp = new RegExp(
+  resourceTypeToKey.environment.toLowerCase() + '-\\w{8}-\\w{4}-\\w{4}-\\w{4}-\\w{12}'
```

Comment:
I'm pretty ignorant when it comes to regexes: Is it possible to reuse `uuidRegExp` here instead of writing it again or do we need the escape characters?

Reply:
Made a fix for this. I'm going to store `-\\w{8}-\\w{4}-\\w{4}-\\w{4}-\\w{12}` as a string that we can reuse when we need to create a UUID regExp.